packages/dataviews/src/filter-summary.js

```diff
@@ -62,127 +56,140 @@ const FilterText = ( { activeElement, filterInView, filter } ) => {
 	);
 };
 
-function WithSeparators( { children } ) {
-	return Children.toArray( children )
-		.filter( Boolean )
-		.map( ( child, i ) => (
-			<Fragment key={ i }>
-				{ i > 0 && <DropdownMenuSeparator /> }
-				{ child }
-			</Fragment>
-		) );
+function OperatorSelector( { filter, view, onChangeView } ) {
+	const operatorOptions = filter.operators?.map( ( operator ) => ( {
+		value: operator,
+		label: OPERATORS[ operator ]?.label,
+	} ) );
+	const currentFilter = view.filters.find(
+		( _filter ) => _filter.field === filter.field
+	);
+	const value = currentFilter?.operator || filter.operators[ 0 ];
+	return (
+		<HStack
+			spacing={ 2 }
+			justify="flex-start"
+			className="dataviews-filter-summary__operators-container"
+		>
+			<FlexItem className="dataviews-filter-summary__operators-filter-name">
+				{ filter.name }
+			</FlexItem>
+			{ operatorOptions.length > 1 && (
+				<SelectControl
+					label={ filter.name }
+					value={ value }
+					options={ operatorOptions }
+					onChange={ ( newValue ) => {
+						const newFilters = currentFilter
+							? [
+									...view.filters.map( ( _filter ) => {
+										if ( _filter.field === filter.field ) {
+											return {
+												..._filter,
+												operator: newValue,
+											};
+										}
+										return _filter;
+									} ),
+							  ]
+							: [
+									...view.filters,
+									{
+										field: filter.field,
+										operator: newValue,
+									},
+							  ];
+						onChangeView( {
+							...view,
+							page: 1,
+							filters: newFilters,
+						} );
+					} }
+					size="small"
+					__nextHasNoMarginBottom
+					hideLabelFromVision
+				/>
+			) }
+		</HStack>
+	);
 }
 
-export default function FilterSummary( { filter, view, onChangeView } ) {
-	const filterInView = view.filters.find( ( f ) => f.field === filter.field );
-	const otherFilters = view.filters.filter(
-		( f ) => f.field !== filter.field
+function ResetFilter( { filter, view, onChangeView, addFilterRef } ) {
+	const isDisabled =
+		view.filters.find( ( _filter ) => _filter.field === filter.field )
+			?.value === undefined && filter.isPrimary;
+	return (
+		<div className="dataviews-filter-summary__reset">
+			<Button
+				disabled={ isDisabled }
+				__experimentalIsFocusable
+				__next40pxDefaultSize
+				variant="tertiary"
+				style={ { justifyContent: 'center', width: '100%' } }
+				onClick={ () => {
+					onChangeView( {
+						...view,
+						page: 1,
+						filters: view.filters.filter(
+							( _filter ) => _filter.field !== filter.field
+						),
+					} );
+					// If the filter is not primary and can be removed, it will be added
+					// back to the available filters from `Add filter` component.
+					if ( ! filter.isPrimary ) {
```

Comment:
Here  we transfer focus to the `add filter` button when removing one.

Reply:
I guess the comment here could be:

```
// If the filter is not primary, it will be removed from the view 
// and the popup/search widget closed.
// In that case, move focus back to the Add filter component.
```